extras/3_nfs.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/bash
+
+WINDOWS_IMAGE="http://10.8.120.188/Summit2019.raw"
+#SERVER_IP="192.168.111.1"
+SERVER_IP=$( ip -f inet addr show baremetal |  grep inet | awk '/inet / {print $2}' | cut -d/ -f1)
+VM_SIZE="60"
+VM_MEMORY="8192"
+
+SERVERS=$(oc get nodes -o jsonpath={.items[*].status.addresses[?\(@.type==\"InternalIP\"\)].address} | xargs)
+for server in $SERVERS; do
+   ssh core@$server "sudo bash -c 'curl http://karmatron.mooo.com/cnv-bridge > /var/lib/cni/bin/cnv-bridge' ; sudo chmod u+x /var/lib/cni/bin/cnv-bridge"
+done
+
+yum -y install nfs-utils
+mkdir /windows
+echo "/windows *(rw,no_root_squash)"  >>  /etc/exports
+chcon -t svirt_sandbox_file_t /windows
+chmod 777 /windows
+exportfs -r
+systemctl start nfs ; systemctl enable nfs-server
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: windows
+spec:
+  capacity:
+    storage: ${VM_SIZE}Gi
+  accessModes:
+  - ReadWriteMany
+  nfs:
+    path: /windows
+    server: ${SERVER_IP}
+  persistentVolumeReclaimPolicy: Recycle
+EOF
+cd /windows
+curl ${WINDOWS_IMAGE} > disk.img
```

Comment:
curl has a -o option to set the output file which should prevent you from needing to use cd /windows.  Also we'll be able to see transfer progress while downloading the large image file.